### PR TITLE
Setup Lido Partner LM + July Payloads

### PR DESCRIPTION
**Pull request by Tritium-VLK**
Lido has provided 30k $ARB on Arbitrum and 2.47 wstETH on Optimism in order to incentivise on Balancer in July. 

The funds were sent to a new safe in order to keep accounting separate for STIP.  The Lido funds will be handled by the Multisig with the address [0xeD2Bc4e72Ed7D61FA4812a6B0E3d16F8Ad8369a1](https://github.com/BalancerMaxis/bal_addresses/blob/main/extras/multisigs.json#L41), which has been deployed on Mainnet, Op and Arbitrum so far.  The safe is currently owned by the [Balancer Maxis Signer Set](https://github.com/BalancerMaxis/bal_addresses/blob/2f8e73c168030fa4a2c7bebd09cc177665ef8252/extras/signers.json#L2) and was verified as such in [here](https://github.com/BalancerMaxis/bal_addresses/pull/357).

The following [Google Sheet](https://docs.google.com/spreadsheets/d/19nvMe95WywXBl6DzKrzlVVf3CNgPZHLQT30IIVbGWiU/edit?gid=0#gid=0) is being used to coordinate the plan with Lido.

This PR includes some changes to the Hiddenhand Payload Reporting to cover multichain.  It also includes 3 payloads:

ARB/2024-07-04.json: Bribes for the next 2 weeks on Arbitrum Column G on the Arbi July tab of the sheet.
ARB/2024-07-05-aura-direct.json: First week of payments for 2 weeks of Aura direct on Arbi (Column E), this will need to be rerun in 1 week for Column F
OP/2024-07-04.json: 4 weeks of Aura incentives using the wstETH as described on the Op July tab.  Note the 2 periods, which are each 2 week aura epochs. 

Note that an Earmark Rewards will have to be called on each of the pools after executing the transfer to direct ARB/2024-07-05.

### CODE REVIEW

**Comment by Tritium-VLK** [comment]
Note that extra stash addresses can be checked by going to the Aura UI, finding the pool, and looking up the aura stash in dev mode.  Here is an example for [ezETH[(https://app.aura.finance/#/42161/pool/54).  Also note that I added a description in directly in the ARB/2024-07-05.json file describing which pool each one was for to make verification easier.  @gosuto-inzasheru @jalbrekt85 maybe we should consider adding Aura ExtraStashes to bal_addresses/our reversebook. 

image

**Comment by franzns** [comment]
OP payload lgtm

File: MaxiOps/00partnerLM/LIDO/ARB/2024-07-04.json [hunk 1]
```diff
@@ -0,0 +1,169 @@
+{
+  "version": "1.0",
+  "chainId": "42161",
+  "createdAt": 1698450039804,
+  "meta": {
+    "name": "",
+    "description": "July OP Bribs from Lido",
```
[the remark this answers is not in the document]
Comment (Tritium-VLK) [reply MaxiOps/00partnerLM/LIDO/ARB/2024-07-04.json:7]: Yes just a typo in description.  Don't even think I need to change that.

**Comment by Tritium-VLK** [comment]
Loading: 

OP/2024-07-04:[Nonce 1 on Op Safe](https://app.safe.global/transactions/tx?id=multisig_0xeD2Bc4e72Ed7D61FA4812a6B0E3d16F8Ad8369a1_0x73456998ddb95405a4d53f276e02635002ca2b2089fa8a320dfefb4e3938a396&safe=oeth:0xeD2Bc4e72Ed7D61FA4812a6B0E3d16F8Ad8369a1)

ARB/2024-07-04: [Nonce 0 on Arbi Safe](https://app.safe.global/transactions/queue?safe=arb1:0xeD2Bc4e72Ed7D61FA4812a6B0E3d16F8Ad8369a1)

ARB/2024-07-05-arb-direct: [Nonce 1 on Arbi Safe](https://app.safe.global/transactions/queue?safe=arb1:0xeD2Bc4e72Ed7D61FA4812a6B0E3d16F8Ad8369a1). ***Note that these pools will need an earmark on Aura before yields show up.  Can do manually or will happen eventually.  As they may not have BAL rewards could take a long time for an autoearmark***

Also used Den to schedule another run of the transfers 2024-07-05 next week.

File: MaxiOps/00partnerLM/LIDO/ARB/2024-07-04.json [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+        ],
+        "name": "depositBribe",
+        "payable": false
+      },
+      "contractInputsValues": {
+        "_proposal": "0x9211c605c0a7fa383d5bb82f1c30a0fa01dc7a5293ab31d7bac36d29d84a540f",
+        "_token": "0x912CE59144191C1204E64559FE8253a0e49E6548",
+        "_amount": "3750000000000000000000",
+        "_maxTokensPerVote": "0",
+        "_periods": "1"
```
[the remark this answers is not in the document]
Comment (Tritium-VLK) [reply MaxiOps/00partnerLM/LIDO/ARB/2024-07-04.json:79]: Correct.  Have to do 2 separate bribs on Arbi, becuase the second 2 weeks is larger due to no direct incentives.

**Comment by Zen-Maxi** [comment]
Reviewed each payload and transaction in den. These transactions will execute the desired outcome as described above by Tritium's text and google sheet. All 3 are signed 3/3 ready to exec.

**Review by Zen-Maxi — APPROVED** [review]

**Pull request merged**